dump/dump.c

```diff
@@ -64,26 +64,34 @@ static unsigned int exfat_count_used_clusters(unsigned char *bitmap,
 	return count;
 }
 
+
+static off_t c2o(off_t heap_offset, unsigned int clu, unsigned int clu_size)
```

Comment:
we don't need to prefix(exfat_) for this function ?

Reply:
This function is static, So I don't append the prefix "exfat_".

Reply:
Do whatever you want. but all function has exfat prefix.